geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/analytics/HistogramJob.scala

```diff
@@ -96,16 +101,26 @@ class HistogramJob(args: Args) extends GeoMesaBaseJob(args) {
   val aggregates: UnsortedGrouped[Product, Long] = groups.aggregate(Aggregator.size)
 
   // write out the histogram to a tsv file - we create the tabs ourselves since length isn't known
-  aggregates.toTypedPipe.map { case (group, count) => s"${group.productIterator.mkString("\t")}\t$count" }
+  if (writeToAccumulo) {
+    aggregates.toTypedPipe
+      .flatMap { case (group, count) =>
+      val mut = new Mutation(new Text(feature))
+      mut.put(EMPTY_COLF, new Text(s"${group.productIterator.mkString("\t")}"), new Value(count.toString.getBytes))
+      Seq(mut).map((null: Text, _))
```

Comment:
so i think for the stats use case, the group will be just the attribute we are histogramming. so how about:
row: feature~attribute~group.last
cf: group(0 to n-2) as a string (maybe ~ separated to be consistent with our other schemas?) - will usually be empty
cq: count
value: empty
I think we would also want things like:
feature~attribute::cardinality::<cardinality>
feature~attribute::min::<min>
feature~attribute::max::<max>
but those might be for a different PR